src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -3978,21 +3986,9 @@ gregtech.multiblock.multiple_recipemaps.tooltip=Screwdriver the controller to ch
 gregtech.multiblock.multiple_recipemaps_recipes.tooltip=Machine Modes: §e%s§r
 gregtech.multiblock.multiple_recipemaps.switch_message=The machine must be off to switch modes!
 
-gregtech.multiblock.preview.zoom=Use mousewheel or right-click+Drag to zoom
+gregtech.multiblock.preview.zoom=Use mousewheel or right-click + drag to zoom
 gregtech.multiblock.preview.rotate=Click and drag to rotate
-gregtech.multiblock.preview.any_hatch=Any tier hatch can be used to form the structure
-gregtech.multiblock.preview.any_hatch_maintenance=Any type of maintenance hatch can be used to form the structure
-gregtech.multiblock.preview.limit=Minimum Limit: %d
-gregtech.multiblock.preview.limit_per_layer=Only %d per layer
-gregtech.multiblock.preview.only=Only %s allowed
-gregtech.multiblock.preview.only_location=Only allowed at %s
-gregtech.multiblock.preview.location.b_c=Bottom Center
-gregtech.multiblock.preview.location.b_l=Bottom Layer
-gregtech.multiblock.preview.location.t_c=Top Center
-gregtech.multiblock.preview.primitive_pump.hatch=Only Pump Hatch, ULV Hatch, or LV Hatch allowed
-gregtech.multiblock.preview.location_end=Very End
-gregtech.multiblock.preview.distillation_multi_fluid=Quadruple/Nonuple Output Hatches not allowed
-gregtech.multiblock.preview.clear_amount=Must have a clear %dx%dx%d space in front
```

Comment:
This preview comment could come back, for the Large Combustion Engine intakes, and the Large Turbine rotor holders